.github/workflows/test.yml

```diff
@@ -243,6 +243,48 @@ jobs:
           java --version
           native-image --version
         if: runner.os == 'Windows'
+  test-liberica:
+    needs: test
+    name: Liberica (${{ matrix.java-version }}, '${{ matrix.version }}', ${{ matrix.os }})
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        java-version: ['17', '21.0.2']
+        version: ['', 'std', 'full']
+        os: [ubuntu-latest, macos-latest, windows-latest]
+    steps:
+      - uses: actions/checkout@v4
+      - name: Run setup-graalvm action
+        uses: ./
+        with:
+          distribution: liberica
+          java-version: ${{ matrix.java-version }}
+          version: ${{ matrix.version }}
```

Comment:
We actually am phasing out `version` in favor of just the `java-version`. It seems there are different variants of Liberica. Can these also be pulled by `setup-java` and if yes, how?

Reply:
AFAICT this is controlled by the `java-package` parameter:

> java-package: The packaging variant of the chosen distribution. Possible values: jdk, jre, jdk+fx, jre+fx. Default value: jdk

The value of `jdk` corresponds to `version: std`, and `jdk+fx` corresponds to `version: full`

Reply:
Thanks! Mapping `jdk+fx` to `full` seems like a hack, assuming fx stands for JavaFX? If Liberica users are more familiar with `std` and `full`, I'm ok with using those directly, maybe also allowing `jdk` and `jdk+fx` for full compatibility with `setup-java`. In any case, we should add [the `java-package` input from `setup-java`](https://github.com/actions/setup-java/blob/9704b39bf258b59bc04b50fa2dd55e9ed76b47a8/action.yml#L13-L16) and not use the `version` input for this.

Reply:
Ok, I've added the `java-package: jdk | jdk+fx` input